pyvo/registry/rtcons.py

```diff
@@ -539,12 +547,11 @@ class Spatial(Constraint):
         >>> resources = registry.Spatial((SkyCoord("23d +3d"), 3))
     """
     _keyword = "spatial"
-    _condition = "1 = CONTAINS({geom}, coverage)"
     _extra_tables = ["rr.stc_spatial"]
 
     takes_sequence = True
 
-    def __init__(self, geom_spec, order=6):
+    def __init__(self, geom_spec, intersect="covers", order=6):
```

Comment:
This added an unnoticed backward incompatible API change. I'll open a PR to remedy this before rolling out v1.5, and ultimately #507 will address any similar future issues.

Reply:
Oops sorry

Reply:
I'm sorry, this should be something to spot during the review. But again, a prime example of why I'm pushing for #507 as we shouldn't really be catering for the sole usage of positional arguments for everything.